eos/statistics/log-prior.cc

```diff
@@ -619,6 +619,74 @@ namespace eos
                     return true;
                 }
         };
+
+        /*!
+         * Poisson prior distribution
+         */
+        class Poisson :
+            public LogPrior
+        {
+            private:
+                Parameter _parameter;
+
+                std::string _name;
+
+                const double _k;
+
+                const double _ln_norm;
+
+            public:
+                Poisson(const Parameters & parameters, const std::string & name, const double & k) :
```

Comment:
I think we might have made a mistake here. The distribution should be a function of $k$ with $\lambda$ as a parameter. I think we accidentally switched the roles of the two. And the connection to the Gamma distribution should be $Poisson(k;\lambda) = Gamma(\lambda; a=k+1, b=1)$.
But maybe I confused something here.